interop/authzen-api-gateways/test-harness/test/runner.ts

```diff
@@ -6,12 +6,12 @@ const AUTHZEN_PDP_URL =
 const AUTHZEN_PDP_API_KEY = process.env.AUTHZEN_PDP_API_KEY;
 
 enum OutputTypes {
-  MARKDOWN,
+  HTML,
```

Comment:
the output is actually markdown with embedded HTML. I don't think this change makes sense (at least to me)